README.asciidoc

```diff
@@ -194,7 +194,17 @@ Elasticsearch uses https://gradle.org[Gradle] for its build system.
 
 In order to create a distribution, simply run the `./gradlew assemble` command in the cloned directory.
 
-The distribution for each project will be created under the `build/distributions` directory in that project.
+To build distributions from your local Elasticsearch repo, run:
```

Comment:
FWIW, you can now run `./gradlew localDistro` which will build the appropriate distribution for the current platform. The task also prints the output location for you so you don't have to guess about searching for it. We should probably mention it here as it's the preferred way to locally build Elasticsearch and get up and running quickly.

Reply:
Updated, even better! Now its really clear how to get it up and running